diff '--color=auto' -ruN ../../../multitor/src/helpers ./src/helpers
--- ../../../multitor/src/helpers	2022-01-31 19:54:34.185464759 +0000
+++ ./src/helpers	2022-01-31 20:11:42.309448765 +0000
@@ -686,9 +686,6 @@
 
   fi
 
-  # Flush etc/ directory.
-  rm -fr "${_etc:?}"/*
-
   return $_STATE
 
 }
diff '--color=auto' -ruN ../../../multitor/src/__init__ ./src/__init__
--- ../../../multitor/src/__init__	2022-01-31 19:54:34.185464759 +0000
+++ ./src/__init__	2022-01-31 21:06:46.247926702 +0000
@@ -562,11 +562,6 @@
 
       _proxy_ports=()
 
-    else
-
-      # Flush etc/ directory.
-      rm -fr "${_etc:?}"/*
-
     fi
 
     for i in $(seq 0 $((init_number - 1))) ; do
@@ -749,9 +744,6 @@
 
     done
 
-    # Flush etc/ directory.
-    rm -fr "${_etc:?}"/*
-
     # shellcheck disable=SC2034
     # Manages proxy processes stack.
     for _proc in "haproxy" "${_proxy_list[@]}" ; do
